bootstrap/lib/mix/tasks/nerves/new.ex

```diff
@@ -3,7 +3,11 @@ defmodule Mix.Tasks.Nerves.New do
   import Mix.Generator
 
   @nerves Path.expand("../../../..", __DIR__)
-  @version "0.2"
+
+  @nerves_vsn "0.4.0"
```

Comment:
Is there a reason we're constraining the `nerves` version to the minor release, but `bootstrap` and `runtime` only to the major version? I'm thinking most people would likely only want bugfixes/patches applied automatically to their projects, as opposed to minor/feature releases.